foundry/entities/actor.d.ts

```diff
@@ -27,14 +28,15 @@
  *
  * @example <caption>Retrieve an existing Actor</caption>
  * ```typescript
- * if (game.actors === undefined) throw "Too early to use an enitiy collection";
+ * if (!game.actors) throw new Error("Too early to use an entity collection");
  * let actor = game.actors.get(actorId);
  * ```
  */
 declare class Actor<
   D extends Actor.Data = Actor.Data,
-  I extends Item<Actor.OwnedItemData<D>> = Item<Actor.OwnedItemData<D>>
-> extends Entity<D> {
+  I extends Item<Actor.OwnedItemData<D>, any> = Item<Actor.OwnedItemData<D>>,
+  PD extends D = D
```

Comment:
I put this as the 3rd parameter because this way it's backwards compatible and people who don't care about it can just omit the parameter.